### PR TITLE
BaSP-TG-78: Styles fixes and dark mode

### PR DESCRIPTION
**Pull request by Baldizan**
### Implemented dark mode and styles fixes.

> **TG-77: Table component inconsistent space** [Bug]

Due to the low amount of information shared on the Tasks table, the Edit and Delete buttons tend to move, varying their position from page to page. Now they always appear at the end of the table:
imagen

> **TG-81: Number of page not centered** [Style fix]

Due to the "See Employees" button within the Project's table, rows tend to be higher than other entities' pages. This caused the pagination button to vary its position. Table's height was modified, triggering the button to stay on the same place regardless of the height variation. 

_Page 1:_
imagen

_Page 2:_
imagen

> **TG-82: Contrast colors of green buttons and red alerts** [Style fix]

Inputs error messages did not pass the QA contrast check. 
The font weight was modified, as well as the letter spacing for better readability. 

imagen

> **TG-83: Messages should be bigger and centered** [Style fix]

Success and Error Modals content was not aligned with the close button, and their font size was smaller than supposed to.

imagen

> **TG-78: Dark mode** 

- To test Dark Mode, go to your browser preferences and choose Dark Mode.
imagen

For learning purposes, I created a Dark Mode, which detects the User's browser preferences, and sets the Theme (dark/light) accordingly. 
This feature could be impproved in the future, by providing the user with the possibility to modify the default Theme difrectly from their profile's settings. 

imagen

imagen

imagen

imagen

### CODE REVIEW

**Review by LucianaPrat — APPROVED** [review]

File: src/Components/Header/header.module.css [hunk 1]
```diff
@@ -36,11 +36,11 @@
   height: 50px;
   right: 0;
   border-radius: 0 0 0 10px;
-  border-bottom: 1px solid #373867;
-  border-left: 1px solid #373867;
-  background-color: #f0f4f9;
+  border-bottom: 1px solid #d7d7e1;
```
Comment (elias-92) [inline src/Components/Header/header.module.css:39]: To this color (#d7d7e1) you could put it in a variable.

File: src/Components/Shared/Input/input.module.css [hunk 2]
```diff
@@ -20,28 +20,29 @@
...
   cursor: pointer;
+  background-color: var(--primaryColorLighter);
 }
 
 .input:focus {
   background-color: #fff;
   cursor: auto;
-  outline: 1px solid #373867;
+  outline: 1px solid var(--primaryColor);
+  background-color: #fff;
```
Comment (elias-92) [inline src/Components/Shared/Input/input.module.css:40]: To this color (#fff) you could put it in a variable and it appears repeated, be careful.

File: src/Components/Shared/Modal/modal.module.css [hunk 3]
```diff
@@ -1,18 +1,10 @@
...
-  top: 0;
-  left: 0;
-  position: fixed;
-  display: flex;
-  justify-content: center;
-  align-items: center;
-  height: 100%;
-  width: 100%;
-  background-color: rgba(0, 0, 0, 0.5);
+  box-shadow: 0 0 0 500vmax rgb(0 0 0 / 0.5);
```
Comment (elias-92) [inline src/Components/Shared/Modal/modal.module.css:2]: You could change the rgb to hex.

**Review by elias-92 — APPROVED** [review]

**Review by mr-valen — COMMENTED** [review]
Superadmin modal appears at top instead of center.
Navbar anchors have a different color.

**Review by mr-valen — APPROVED** [review]
Everything looks great, I'm in love with dark mode 😍

**Pull request merged**